packages/markdown-it-synapse/package.json

```diff
@@ -1,9 +1,16 @@
 {
   "name": "markdown-it-synapse",
-  "version": "1.1.13",
+  "version": "2.0.0",
```

Comment:
The UMD output is changing because limitations in esbuild prevent us from creating a bundle that has the same export shape, so let's just bump to 2.0.0